components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java

```diff
@@ -227,6 +244,18 @@ public void handle(HttpServletRequest request, HttpServletResponse response) thr
                                     "Request Headers: " + getHeaderString(request) + "\n" +
                                     "Thread Id: " + Thread.currentThread().getId());
                         }
+                        if (LoggerUtils.isDiagnosticLogsEnabled()) {
+                            Map<String, Object> params = new HashMap<>();
+                            params.put("Context id", context.getContextIdentifier());
+                            params.put("Originating address", request.getRemoteAddr());
+                            params.put("Request Headers", getHeaderString(request));
+                            params.put("Thread Id", Thread.currentThread().getId());
+                            LoggerUtils.triggerDiagnosticLogEvent(
+                                    FrameworkConstants.LogConstants.AUTHENTICATION_FRAMEWORK, params,
+                                    LogConstants.FAILED, "Same context is currently in used by a " +
```

Comment:
again better to reword to something like "same authentication request is being processed by another thread. Could be a possible double submit or replay request"

Reply:
Addressed with https://github.com/wso2/carbon-identity-framework/pull/4326/commits/c15402ecaefef87740033c886e79165563e32b8f